server/dummyData/dummyAdmins.js

```diff
@@ -1,5 +1,4 @@
-const { ObjectID } = require('mongodb');
-
+const ObjectID = require('mongodb').ObjectID;
```

Comment:
You modified this from the previous line which destructured. I thought we decided to destructure when we're importing modules (it's what most of the rest of the codebase does)

Reply:
Not sure why that happened. Fixed.